package.json

```diff
@@ -39,6 +39,12 @@
     "build:assets:other": "cpy --flat 'src/assets' '!**.svg' '!**.tsx' assets",
     "build:assets:svg": "svgo --folder src/assets -o assets",
     "build:react": "tsup src/react/index.ts --dts --format=esm,cjs --sourcemap --out-dir=react",
+    "build:src": "yarn run build:src:logo && yarn run build:src:favicon:png && yarn run build:src:favicon:ico && yarn run build:src:android-icon && yarn run build:src:apple-touch-icon",
```

Comment:
I see what you're going for here! Typically the verb “generate” (sometimes “codegen”) is used to convey that a script is creating source code rather than build artifacts. Let's rename the `build:src:*` scripts to `generate:*`.

Reply:
Great feedback! I'll keep this in mind going forward.